client/dashboard/app/queries.ts

```diff
@@ -111,6 +115,34 @@ export function siteWordPressVersionMutation( siteSlug: string ) {
 	};
 }
 
+export function siteResetContentSummaryQuery( siteIdOrSlug: string ) {
+	return {
+		queryKey: [ 'site-reset-content', siteIdOrSlug ],
+		queryFn: () => fetchSiteResetContentSummary( siteIdOrSlug ),
+	};
+}
+
+export function resetSiteMutation( siteIdOrSlug: string ) {
+	return {
+		mutationFn: () => resetSite( siteIdOrSlug ),
+	};
+}
+
+export function siteResetStatusQuery( siteIdOrSlug: string ) {
+	return {
+		queryKey: [ 'site-reset-status', siteIdOrSlug ],
+		queryFn: () => fetchSiteResetStatus( siteIdOrSlug ),
+		refetchInterval: ( query: Query< SiteResetStatus > ) => {
```

Comment:
Does this need to be the default behavior of the reset site status query? It seems like we only need this functionality in the component that needs to track the refetch status.

Reply:
Interesting question! Indeed, it's not needed for simple sites. I guess we could conditionally add it in the component when we have an atomic site. ...thinking...

Reply:
Yup, I think we can do something like this:

```js
useQuery< SiteResetStatus >( {
  ...siteResetStatusQuery( site.ID ),
  ...site.is_wpcom_atomic ? ( { refetchInterval: ... } ) : ( {} )
} );
```